Sources/Sentry/SentryHub.m

```diff
@@ -185,6 +197,51 @@ - (SentrySession *)incrementSessionErrors
     return sessionCopy;
 }
 
+- (void)captureCrashEvent:(SentryEvent *)event
+{
+    SentryClient *client = [self getClient];
+
+    @synchronized (_crashEventAndSessionLock) {
+        //  When enableAutoSessionTracking is enabled, we send the crash event and session together.
+        if (nil != client && [client.options.enableAutoSessionTracking boolValue]
```

Comment:
I added a test for this: testCaptureCrashEvent_SessionExistsButAutoSessionTrackingDisabled

Reply:
I just found out, that if you just enabled auto session tracking and you have a hard crash on the disk. We won't send the hard crash, because we don't have an existing session. I'm working on solving this problem now.